drivers/overlay/overlay.go

```diff
@@ -431,6 +441,24 @@ func (d *Driver) Get(id string, mountLabel string) (s string, err error) {
 		return "", err
 	}
 
+	newlowers := ""
+	for _, lower := range strings.Split(string(lowers), ":") {
+		if _, err = os.Stat(string(lower)); err != nil {
+			for _, p := range d.AdditionalImageStores() {
+				lower = path.Join(p, d.name, lower)
+				if _, err := os.Stat(lower); err != nil {
+					return "", err
+				}
+				break
```

Comment:
If AdditionalImageStores has more than one element, will this ever search past the first one?